arm/Microsoft.EventGrid/systemTopics/deploy.bicep

```diff
@@ -143,16 +143,6 @@ resource systemTopic_diagnosticSettings 'Microsoft.Insights/diagnosticsettings@2
   scope: systemTopic
 }
 
-module systemTopic_privateEndpoints '.bicep/nested_privateEndpoint.bicep' = [for (privateEndpoint, index) in privateEndpoints: if (!empty(privateEndpoints)) {
```

Comment:
Reference #1386 in description as this solves the issue

Reply:
Linked